### PR TITLE
Tagline and font changes to the welcome page

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

**Comment by jacksonh** [comment]
@satindar ok to merge this one too?

**Comment by satindar** [comment]
> @satindar ok to merge this one too?

Yeah go for it. I'll wait until we both merge before creating the TF build

**Review by satindar — APPROVED** [review]

**Pull request merged**
